Resources/Prototypes/_SpaceCats/Entities/Clothing/OuterClothing/vests.yml

```
@@ -0,0 +1,28 @@
+        Slash: 0.7
+        Piercing: 0.7
+        Heat: 0.8
+        Radiation: 0.9
+        Caustic: 0.9
+  - type: Tag
+    tags:
+    - WhitelistChameleon
+    - HighRiskItem
+  - type: ClothingGrant
+    component:
+    - type: DamageTeleport
```

yamllint (1.29.0-1)

[error] no new line character at the end of file

(new-line-at-end-of-file)